tests/test_api/test_products.py

```diff
@@ -25,6 +29,20 @@ async def test_product_create(self, product_create_payload):
         )
         assert response.status_code == self.status_codes.HTTP_200_OK, message
 
+    async def test_product_remove(
```

Comment:
we already have the similar test, it's `test_user_product_remove`

Reply:
That's not mine